vipul/Gallery_System/contact.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en-US">
+	<head>
+		<meta name="description" content="This is webpage for online photo gallery system.">
+		<meta name="keyword" content="online,photo,gallery.">
+		<title>Photo Gallery | Contact Us</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+	<body>
+		<div class="container">
+			<div class="header">
+				<h1>
+					<a href="index.html">Photo Gallery</a>
+				</h1>
+			</div>
+			<div class="list">
+				<ul class="nav">
+					<li><a href="index.html" class="active">HOME</a></li>
+					<li class="sublist">
+						<a href="wild.html">CATEGORY</a>
+						<ul class="dropdown">
+							<li><a href="wild.html">Wildlife</a></li>
+							<li><a href="landscape.html">Landscape</a></li>
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="main-content-contact-us">
+			<p>Contact Us</p>
+			<div class="contact-form">
+				<form>
+					<fieldset> 
+						<p style="font-size:25px;">Full Name : <input type="text" name="name"></p>
+						<p style="font-size:25px;">Email : <input type="email" name="user_email"></p>
+						<p style="font-size:25px;">Subject : <input type="text" name="subject"></p>
+						<p style="font-size:25px;">Message : <textarea rows="5"></textarea></p>
+						<center><input type="submit" value="Send Your Message"></center>
+					</fieldset>
+				</form>
+			</div>
+			<div class="contact">
+				<div class="contact-us">
+					<h3>Address:</h3>
+					<p>Pune, Maharashtra,</p>
+					<p>India</p>
+					<br>
+				</div>
+				<div class="contact-us">
+					<h3>Phone:</h3>
+					<p style="color:#008CBA;font-size:20px;">+91 7030791319</p>
+					<br>
+				</div>
+				<div class="contact-us">
+					<h3>Email:</h3>
+					<p style="color:#008CBA;font-size:20px;">vipulsuryawanshi28@gmail.com</p>
+					<br>
+				</div>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes